doc/material-migration.md

```diff
@@ -1,4 +1,332 @@
-﻿# Updating to Uno.Material v2
+﻿---
+uid: uno.themes.material.migration
+---
+
+# Updating to Uno.Material v3
+
+Version 3.x of Uno.Material introduces support for [Lightweight Styling](lightweight-styling.md) as well as some breaking changes to the default style keys for some controls. The tables below provide details on the changes that have been made to the resource keys within Uno.Material.
```

Comment:
```suggestion
Uno.Material v3 (not to be confused with Material 3 from Material Design) introduces support for [Lightweight Styling](lightweight-styling.md) as well as some breaking changes to the default style keys for some controls. Refer to the document below for the changes that have been made within Uno.Material.
```

Reply:
i would move ##Styles and ##Resources each to a separate file, and link them here
these two files, dont need to be added to the toc.yml

Reply:
I feel like we should keep them here as a single spot for all migration information. If we move them then I just create `material-migration-v3-styles.md` and `-v3-resources.md` pages?

I'm worried people will miss the relevant information if they just quickly scan this migration doc and don't see the information they are looking for,